### DISCUSSION TITLE
Distinguish between Fork and Repositories on My Profile page

### DISCUSSION THREAD

**Discussion by mrleblanc101** [community/community#43013] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,
I would like a way to distinguish, or seperate Forks from my own Repositories.
Sometime I make small contributions, fix in documentation, etc in Repo I don't own, but since I need to fork them to open a PR, they become stale and polute my list of Repo. I can become hard to keep your list of repo clean since there is no way to know if all your PRs for this fork have been merged other than to check them all one-by-one.

**Comment by LuisContrerasGlz** [comment]
You can use the "fork" tag in your repository list to filter for only forked repositories. To do this, go to your repository list and enter "is:fork" in the search bar. This will show you a list of all forked repositories in your account.

Alternatively, you can use the "is:source" tag to filter for only repositories that are not forks. This will show you a list of all repositories in your account that are not forks of someone else's repository.

You can also use these tags in combination with other filters to narrow down your search further. For example, you can search for "is:fork language:python" to find all of your forked Python repositories.

**Reply by mrleblanc101** [reply]
Yeah I just discovered the `source` filter, the name is not very clear.
The problem is that only affect my view, not people see my profile.
So I end up pinning every in `source`.

**Reply by LuisContrerasGlz** [reply]
Oh nice, glad to know it is good know, if my answer helped some way could you help me marking it as the answer? haha I am trying to get the galaxy brain in github https://github.com/community/community/discussions/28656

**Comment by ShravanDalavi** [comment]
That would be a good addition I think. Many forked repositories go dormant after a contribution has been merged, but they still pollute the profile and make it more difficult to feature personal projects. Having a separate section or option to filter out forks from original repos would make profiles much cleaner while still preserving contribution history. This can be especially helpful for developers who contribute to many open source projects. 👍
